### PR TITLE
Update permissions-deployment-schema.json

### PR DESCRIPTION
**Pull request by athmangude**
### Changed
- Set scheme to enum
- Set type of permission member to array
- Moved permission environment to definitions

### CODE REVIEW

**Comment by athmangude** [comment]
@darrelmiller and @wandeg, please review this PR merging into [microsoftgraph:migratespecs](https://github.com/microsoftgraph/kibali/tree/migratespecs)

File: specs/permissions-deployment-schema.json [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
+          "id": {
+            "type": "string"
+          },
+          "resourceAppId": {
+            "type": "string"
+          },
+          "environment": {
+            "type": "string"
+          },
+          "scheme": {
```
Comment (darrelmiller) [inline specs/permissions-deployment-schema.json:32]:
```suggestion
          "scheme": {
            "type": "string",
```

File: specs/permissions-deployment-schema.json [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
+      "properties": {
+        "properties": {
+          "id": {
+            "type": "string"
+          },
+          "resourceAppId": {
+            "type": "string"
+          },
+          "environment": {
+            "type": "string"
```
[the remark this answers is not in the document]
Comment (darrelmiller) [reply specs/permissions-deployment-schema.json:30]:
```suggestion
            "type": "string",
            "enum": ["public","PPE","fairfax","mooncake","USSec","USNat"]
```

File: specs/permissions-deployment-schema.json [hunk 3]
```diff
@@ -1,39 +1,46 @@
...
+          },
+          "scheme": {
+            "enum": ["Application", "DelegatedPersonal", "DelegatedWork"]
```
Comment (darrelmiller) [inline specs/permissions-deployment-schema.json:33]:
```suggestion
            "enum": ["application", "delegatedPersonal", "delegatedWork"]
```

**Review by darrelmiller — CHANGES_REQUESTED** [review]
I made some suggestions to address James' comments.

**Pull request closed without merging**